### ISSUE TITLE
Error in SwiftyMocky setup command

### ISSUE THREAD

**Issue by Sob7y** [MakeAWishFoundation/SwiftyMocky#217]
Hi, I am facing this issue after updating to Catalina OS

`This copy of libswiftCore.dylib requires an OS version prior to 10.14.4.
`
Is there any solution besides downgrading my OS if this is the cause mainly.

**Comment by amichnia** [comment]
@Sob7y thanks, will be fixed hopefully in next release.

**Comment by amichnia** [comment]
@Sob7y  can you confirm it is still happening after installation of SwiftyMocky 3.3.4 ? I cannot reproduce it, I'm on Catalina (macOS 10.15.1)

**Comment by Sob7y** [comment]
Hello,

I'm sorry, I'm still getting this error:

swiftymocky setup

This copy of libswiftCore.dylib requires an OS version prior to 10.14.4.


By the way, My OS version is 10.15


Best Regards,

**Comment by amichnia** [comment]
I'm keeping it open for now. I will have some more machines updated to Catalina soon, I will see if 3.4.0 fixed that.

I would also suggest removing SwiftyMocky and installing it via mint again. It looks a bit like you still have a binary compiled with old sdk, but it's hard to verify now.

**Comment by jibeex** [comment]
Hello, I have got this issue and reloved it. I leave the method here to help more people.

For machine graded to latest version of Catalina, SwiftyMocky needs to be built with the new X-Code.

Verified processes:
1. Download X-Code 11.3
2. Set the command-line tool of X-Code 11.3 as the default command-line tool
3. uninstall SwiftyMocky from `Mint`
4. Reinstall `SwiftyMocky` via `Mint` which will rebuild SwiftyMocky with the X-Code 11.3 command-line tool
5. `swiftymocky` command should work now.
